### DISCUSSION TITLE
ASR-unregister configuration server from the vault failed

### DISCUSSION THREAD

**Discussion by saellab01** [microsoft/MicrosoftEdge-Extensions#8] [Q&A]
hello,
my issue is that i already registered a vmware virtual machine as configuration server for my vault and then i had to unregister it, i tried the command :
reg delete "HKLM\Software\Microsoft\Azure Site Recovery\Registration"
net stop dra
the command passed and returned that the ristration was deleted, but when i try to configure this machine with another vault, i discovered that it still knows the old one and wont to be deleted...
i am actually blocked in this step any help please ?
thanks

**Comment by nagachaitanyalokam** [comment]
Hello @saellab01, is your question anyway related to Microsoft Edge Extensions or our Partner center?
